src/drivers/intel/baytrail/ipc.c

```diff
@@ -67,7 +67,8 @@ static void do_notify(void)
 
 	/* copy the data returned from DSP */
 	if (msg->rx_size && msg->rx_size < SOF_IPC_MSG_MAX_SIZE)
-		mailbox_dspbox_read(msg->rx_data, 0, msg->rx_size);
+		mailbox_dspbox_read(msg->rx_data, SOF_IPC_MSG_MAX_SIZE,
```

Comment:
You have changed the number of function parameters, but haven't changed function definition.

Reply:
Changes were in different pr - merged prs